src/showcase/src/showcase.ts

```diff
@@ -621,7 +621,7 @@ const init = () => {
   // This is not very user friendly (in particular we don't show anything like a
   // 404) but this is an dev page anyway.
   const route = window.location.pathname
-    .replace(process.env.BASE_URL ?? "", "")
+    .replace(import.meta.env.BASE_URL ?? "", "")
```

Comment:
> TS2339: Property 'env' does not exist on type 'ImportMeta'.

Can you either:

- copy/paste `env.d.ts` in `showcase/src`
or
- add following to `showcase/tsconfig.json`

```
"compilerOptions": {
    "types": ["vite/client"]
  }
```

Reply:
It works for me; do you get this when running `tsc --noEmit`?

Reply:
Can you try with `tsc`? That's the source of truth; you may have an editor setup issue

Reply:
Same with `tsc`

<img width="1004" alt="Capture d’écran 2023-04-24 à 18 23 29" src="https://user-images.githubusercontent.com/16886711/234057652-fc51ea3d-2ecf-46e5-b2bc-d8033845c7f9.png">